### DISCUSSION TITLE
Theme removed automatically?

### DISCUSSION THREAD

**Discussion by dumbasPL** [eamodio/vscode-amethyst-theme#18] [General]
I've experienced this twice already and today was the second time. I've been using this theme for quite a while now and vscode just randomly decides to remove it. I have it on my synced extensions list and it magically disappeared from it and from all the editors I use. The first time this happened I thought I made a mistake since I was making a lot of configuration changes at the time. But now I haven't touched anything in a while and it just removed itself, again. I don't think it has anything to do with this extension in particular but I haven't observed this behaviour with anything else. Has this happened to anyone else?

**Comment by eamodio** [comment]
I've never had it with this extension, but I quite often get extensions syncing incorrectly most of the time it re-adds something recently removed but I've had it just remove stuff too.
